src/main/java/hng_java_boilerplate/validEmail/EmailComValidator.java

```diff
@@ -0,0 +1,17 @@
+package hng_java_boilerplate.validEmail;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class EmailComValidator implements ConstraintValidator<ValidEmailCom, String> {
```

Comment:
do you need to write your own validator for email? you should be able to achieve the same with @Email annotation from spring validation. The dependency is already in the application